Directory.build.props

```diff
@@ -24,14 +24,19 @@
     <DebugType>full</DebugType>
     <IsTestProject>$(MSBuildProjectName.Contains('UnitTest'))</IsTestProject>
     <IsLibraryProject>$(MSBuildProjectName.Contains('MvvmCross'))</IsLibraryProject>
+    <IsForms>$(MSBuildProjectName.Contains('.Forms'))</IsForms>
   </PropertyGroup>
   
   <ItemGroup Condition="'$(IsLibraryProject)' == 'true'">
-    <PackageReference Include="MSBuild.Sdk.Extras" Version="1.2.2" PrivateAssets="All" />
+    <PackageReference Include="MSBuild.Sdk.Extras" Version="1.3.0" PrivateAssets="All" />

```

Comment:
Please do this update in a separate PR.

Reply:
yep, no probs - it'll need to happen before v3 of XF that's all. I suspect this whole PR will be broken into chunks and then this one abandoned